data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>2nd sentence:</p><p> </p><p>Original:</p><p>For each dataset, you will load the data into a Pandas DataFrame, clean up the columns into a usable format, and then restructure the data into feature and target data columns.</p><p> </p><p>Proposed:</p><p>For each dataset, you will load the data into a Pandas DataFrame, <strong>clean and transform </strong>the columns into a usable format, and then restructure the data into feature and target data columns.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>